src/blockchain_txn_transfer_hotspot_v1.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package helium;
+
+message blockchain_txn_transfer_hotspot_v1 {
+    bytes gateway = 1;
+    bytes seller = 2;
+    bytes buyer = 3;
+    bytes seller_signature = 4;
+    bytes buyer_signature = 5;
+    uint64 hnt_to_seller = 6;
```

Comment:
```suggestion
    uint64 amount_to_seller = 6;
```

Should this be an `amount` to match the others? And also in bones and not hnt?

Reply:
@evanmcc Do you have thoughts about this?